iib/workers/config.py

```diff
@@ -347,6 +361,21 @@ def validate_celery_config(conf: app.utils.Settings, **kwargs) -> None:
                 'variables must be set to valid strings when "IIB_OTEL_TRACING" is set to True.'
             )
 
+def _validate_multiple_opm_mapping(iib_ocp_opm_mapping : Dict[str, str]) -> None:
+    """
+    Validate iib_ocp_opm_mapping config variable
+    :param dict iib_ocp_opm_mapping: the value of iib_ocp_opm_mapping variable
+    :raises iib.exceptions.ConfigError: if the configuration is invalid
+    """
+    if iib_ocp_opm_mapping is not None:
+        if not isinstance(iib_ocp_opm_mapping, dict):
+            raise ConfigError('iib_ocp_opm_mapping must be a dictionary')
+        opms_defined = set(iib_ocp_opm_mapping.values())
+        for opm_version in opms_defined:
+            if not os.path.exists(f'/usr/bin/{opm_version}'):
```

Comment:
I would go with this kind of check:
https://stackoverflow.com/questions/377017/test-if-executable-exists-in-python

Personally, first thing what come to my mind was: https://docs.python.org/3.12/library/shutil.html#shutil.which

I know we are installing opm in our playbooks to /usr/bin but since this is opensource and opm could be put anywhere in $PATH then I would go with `which` command.

Reply:
Updated, this is simpler, Thank you :)